src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonMetadataServices.Converters.cs

```diff
@@ -50,6 +50,22 @@ public static partial class JsonMetadataServices
         public static JsonConverter<DateTimeOffset> DateTimeOffsetConverter => s_dateTimeOffsetConverter ??= new DateTimeOffsetConverter();
         private static JsonConverter<DateTimeOffset>? s_dateTimeOffsetConverter;
 
+#if NET6_0_OR_GREATER
```

Comment:
```suggestion
#if NETCOREAPP
```

See Commit 1 of https://github.com/dotnet/runtime/pull/65847

Reply:
@eerhardt Why not just `#if NET`? Guess it's too late to do something about it, but I'm still curious

Reply:
For versions `5.0`+, `#if NETCOREAPP` and `#if NET` mean the same thing. I'm not 100% sure why @ViktorHofer choose `NETCOREAPP` over `NET`, but the reasons I can think of are:

1. It's more readable and less ambiguous to someone new. `#if NET` _could_ mean NET Framework too, and doesn't specify whether it counts or not. So unless you knew the difference already, it could be confusing.
2. We already call it NETCOREAPP everywhere else, like in `xxx.netcoreapp.cs` files. So sticking with that terminology everywhere seems appropriate.

@ViktorHofer - thoughts?

Reply:
related: https://github.com/dotnet/runtime/issues/43049. Consistency would of course be nice but I don't assume that's possible given the number of contributors and the size of the repository. I don't know if analyzers for preprocessor symbols exists or is even possible.